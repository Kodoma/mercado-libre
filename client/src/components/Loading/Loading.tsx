import { CircularProgress } from '@material-ui/core';
import React from 'react';
import styles from '../Item/Item.module.css';

const Loading = () => {
    return (
        <div className={styles.loadingContainer}>
            <CircularProgress size={40} />
        </div>
    );
};

export default Loading;
