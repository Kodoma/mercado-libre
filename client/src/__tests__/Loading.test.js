import React from 'react';
import { shallow } from 'enzyme';
import Loading from "../components/Loading/Loading";

describe('__tests__ Loading', () => {
    it('should render my Loading component', () => {
        const component = shallow(<Loading />);
        expect(component.getElements()).toMatchSnapshot();
    });
});