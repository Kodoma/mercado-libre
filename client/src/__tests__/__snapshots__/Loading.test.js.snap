// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__tests__ Loading should render my Loading component 1`] = `
Array [
  <div
    className="loadingContainer"
  >
    <WithStyles(ForwardRef(CircularProgress))
      size={40}
    />
  </div>,
]
`;
